suite-common/icons/assets/icons/doubleCheck.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
+  <path stroke="#0078AC" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="m9.25 5.25-5.5 5.5L1 8m14-2.75-5.5 5.5-1.463-1.463"/>
```

Comment:
Don't put the icon here already with the color, use **171717**

Use classic black color not the blue color.

```suggestion
  <path stroke="#171717" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="m9.25 5.25-5.5 5.5L1 8m14-2.75-5.5 5.5-1.463-1.463"/>
```

Reply:
fixed, also  on the info icon